cmd/mesh/manifest.go

```diff
@@ -111,13 +111,6 @@ func genManifests(inFilename string, setOverlayYAML string) (name.ManifestMap, e
 
 // fetchInstallPackageFromURL downloads installation packages from specified URL.
 func fetchInstallPackageFromURL(mergedICPS *v1alpha2.IstioControlPlaneSpec) error {
```

Comment:
so we are removing this functionality for specifying version and install?

Reply:
for 1.3 only. we will discuss this again in 1.4 timeframe.